Pyxspec compatibility and README (#3)
ISDC-integral/osa-python-docker: README.md

## Image Contents

* A latest Python version, currently 3.10.11.
* OSA

> * HEASoft, usually the latest version. Note that some OSA tools related to response matrix rebinning for ISGRI and JEM-X, but also the pipelines are not compatible with heasoft 6.28 and beyond and will cause an error.

Reviewer: ```suggestion
* HEASoft, usually the latest version. Note that some OSA tools related to response matrix rebinning for ISGRI and JEM-X as well as standard ISGRI and JEM-X pipelines are not compatible with HEASoft versions newer than 6.28.
```
